go.mod

```diff
@@ -1,36 +1,36 @@
 module istio.io/tools
 
-go 1.17
+go 1.18
```

Comment:
Was getting issues of go 1.16 would prefer this other module. Updating this to 1.18 fixes this.